app/assets/css/theme.css

```diff
@@ -108,7 +108,8 @@
   --bg-widget-table-color: var(--ui-gray-3);
   --bg-header-color: var(--white-color);
   --bg-hover-table-color: var(--grey-14);
-  --bg-switch-box-color: var(--ui-gray-5);
+  --bg-switch-box-color: var(--ui-gray-6);
+  --bg-switch-box-hover-color: var(--ui-gray-7);
```

Comment:
remove bg-switch-box-hover-color entirely and reference the colors directly? I need to have two vscode windows open to understand what color is applied with the css color vairables